src/legacy/core_plugins/region_map/public/region_map_visualization.js

```diff
@@ -17,20 +17,20 @@
  * under the License.
  */
 
-import 'plugins/kbn_vislib_vis_types/controls/vislib_basic_options';
 import { i18n } from '@kbn/i18n';
-import { BaseMapsVisualizationProvider } from '../../tile_map/public/base_maps_visualization';
 import ChoroplethLayer from './choropleth_layer';
 import { truncatedColorMaps }  from 'ui/vislib/components/color/truncated_colormaps';
 import { getFormat } from 'ui/visualize/loader/pipeline_helpers/utilities';
-import { TileMapTooltipFormatter } from './tooltip_formatter';
-import 'ui/vis/map/service_settings';
 import { toastNotifications } from 'ui/notify';
 
-export function RegionMapsVisualizationProvider(Private, config) {
+import { TileMapTooltipFormatter } from './tooltip_formatter';
+
+// TODO: reference to TILE_MAP plugin should be removed
+import { BaseMapsVisualizationProvider } from '../../tile_map/public/base_maps_visualization';
```

Comment:
This is another thing we could potentially export directly from `tile_map/public`

Reply:
> We could open another PR that exports it (and any other stuff imported from it) from tile_map/public/index if you think it will make it easier to remember later.

Yes, that can be addressed in another PR.  Maybe tile map and region map should be in the same plugin if region map needs a bunch of stuff from tile map but that can be considered in another PR as well

Reply:
@nreese @thomasneirynck Yeah our idea in the new platform migration plans was to have a single `vis_type_maps` plugin that could register both of the maps visualizations since they are sharing some code. But y'all would probably be better judges of the feasibility of that. In the interim we can just keep them shimmed separately like this.